conf/defaults.ini

```diff
@@ -773,3 +773,17 @@ license_path =
 [feature_toggles]
 # enable features, separated by spaces
 enable =
+
+[preferences]
+# Default date format
+date_format = YYYY-MM-DD HH:mm:ss
+# Used by graph and other places where we only show small intervals
+date_format_interval_PT1S = HH:mm:ss
```

Comment:
What does `PT` stand for? I think we're trying to only use lower case characters for settings.

Reply:
They are standard ISO time intervals, and are uppercase in the standard: https://en.wikipedia.org/wiki/ISO_8601#Time_intervals

Reply:
Got it. But all other settings is lowercase so think consistency is more important. You can still refer to standard ISO time intervals in descriptions/docs. HTTP and OAuth is http and oauth in our settings as an example. 

The alternative I had in mind was to postfix `date_format_interval_` with second, minute, hour, day, month, year etc but no biggie.

Reply:
yea, that sounds better

Reply:
changed the interval names to be just seconds, minutes, days, months, years